### PR TITLE
feat(chip): implementation of the Chip component

### PR DESCRIPTION
**Pull request by TotomisPanos**
Other than the Chip component some changes and bug fixes were done on typography and field.

### CODE REVIEW

File: src/components/Typography/Typography.types.ts [hunk 1]
```diff
@@ -16,15 +16,42 @@ export enum TypographyVariants {
...
-  [TypographyVariants.headline1]: { fontFamily: theme.fontFamilyRegular },
-  [TypographyVariants.headline2]: { fontFamily: theme.fontFamilyRegular },
-  [TypographyVariants.headline3]: { fontFamily: theme.fontFamilyRegular },
-  [TypographyVariants.subtitle1]: { fontFamily: theme.fontFamilyRegular },
-  [TypographyVariants.body1]: { fontFamily: theme.fontFamilyRegular },
-  [TypographyVariants.subtitle2]: { fontFamily: theme.fontFamilyRegular },
-  [TypographyVariants.body2]: { fontFamily: theme.fontFamilyRegular },
-  [TypographyVariants.captionSemiBold]: { fontFamily: theme.fontFamilyRegular },
-  [TypographyVariants.captionRegular]: { fontFamily: theme.fontFamilyRegular },
+  [TypographyVariants.headline1]: {
```
Comment (StefChatz) [inline src/components/Typography/Typography.types.ts:19]: These are all the same values, shouldn't they have different fontFamily and fontStyle each?

File: src/components/Chip/Chip.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import {
+  ChipVariants,
+  getChipColor,
+  getChipBackgroundColor,
+} from './Chip.types';
+import { globalTokens } from '../design-tokens';
+import { Typography, TypographyVariants } from '../Typography';
+
+export type ChipProps = {
+  variant: ChipVariants;
```
Comment (StefChatz) [inline src/components/Chip/Chip.tsx:11]: optional prop, and have "primary" as default value

File: src/stories/Chip/Chip.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import { Chip, ChipVariants } from '../../components/Chip';
+import './Chip.stories.css';
+
+export default {
+  title: 'ReactComponentLibrary/Chip',
+  component: Chip,
+} as Meta<typeof Chip>;
+
+export const Variants = () => {
+  return (
```
Comment (StefChatz) [inline src/stories/Chip/Chip.stories.tsx:12]: Have some sort of title for each one, that explains what each is

**Comment by TotomisPanos** [comment]
comment test

**Pull request merged**